recipes/tutorials/set_up_workflows.ipynb

```diff
@@ -0,0 +1,662 @@
+{
```

Comment:
<p>Maybe we could add a link to the actual Basic project setup tutorial from here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/898/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>